build-system/common/update-design-review-issues.js

```diff
@@ -308,38 +317,24 @@ function unpinGithubIssue(token, repo, number) {
 
 /**
  * @param {Date} date
- * @param {number} dayOfWeek
- * @param {number=} weeks
+ * @param {number=} days
  * @return {Date}
  */
-function getNextDayOfWeek(date, dayOfWeek, weeks = 1) {
+function addDays(date, days = 1) {
   const resultDate = new Date(date.getTime());
-  resultDate.setDate(
-    resultDate.getDate() +
-      (weeks - 1) * 7 +
-      ((7 + dayOfWeek - date.getDay()) % 7)
-  );
+  resultDate.setDate(resultDate.getDate() + days);
   return resultDate;
 }
 
 /**
- * @param {Date} date
- * @param {string} startYyyyMmDd
- * @return {Array<string>}
+ * @param {Date} nextDay
+ * @param {Date} start
+ * @return {RotationItemDef}
  */
-function getRotation(date, startYyyyMmDd) {
-  const [year, month, day] = startYyyyMmDd
-    .split('-')
-    .map((n) => parseInt(n, 10));
-  const start = new Date(year, month - 1, day);
-  const dateBeginningOfDay = new Date(
-    date.getFullYear(),
-    date.getMonth(),
-    date.getDate()
-  );
+function getRotation(nextDay, start) {
   const weeks = Math.round(
     // @ts-ignore date calc
-    (dateBeginningOfDay - start) / (7 * 24 * 60 * 60 * 1000)
+    (nextDay - start) / (7 * 24 * 60 * 60 * 1000)
```

Comment:
Refactor here makes the code much easier to read. 

Nit: convert the value to a named variable for additional clarity.